samples/Common.c

```diff
@@ -1,5 +1,6 @@
 #define LOG_CLASS "WebRtcSamples"
 #include "Samples.h"
+// #define MASTER_DATA_CHANNEL_MESSAGE "This message is from the KVS Master"
```

Comment:
Sugest moving this to `Sample.h` instead.

Reply:
Moved #define MASTER_DATA_CHANNEL_MESSAGE and #define VIEWER_DATA_CHANNEL_MESSAGE to Samples.h